<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>61/10/1</td>
</tr><tr>
<td>Started on:</td><td>Fri Oct 24 06:02:03 IST 2025</td>
</tr>
<tr><td>Total time:</td><td>17 seconds (17204 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testNGDemo.TestScenario5.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario5.first_testcase(TestScenario5.java:13)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace889891977", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace889891977'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario5.first_testcase(TestScenario5.java:13)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario9.seventh_testcase(TestScenario9.java:50)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1777745301", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1777745301'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario9.seventh_testcase(TestScenario9.java:50)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [Hello World] but found [Hello]
	at testNGDemo.TestScenario.first_testcase(TestScenario.java:25)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1579957528", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1579957528'><pre>java.lang.AssertionError: The following asserts failed:
	expected [Hello World] but found [Hello]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
	at testNGDemo.TestScenario.first_testcase(TestScenario.java:25)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario3.third_testcase(TestScenario3.java:26)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace216856121", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace216856121'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario3.third_testcase(TestScenario3.java:26)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario5.sixth_testcase(TestScenario5.java:44)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1281414889", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1281414889'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario5.sixth_testcase(TestScenario5.java:44)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario7.fifth_testcase(TestScenario7.java:38)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace634297796", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace634297796'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario7.fifth_testcase(TestScenario7.java:38)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario7.second_testcase(TestScenario7.java:19)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1413623320", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1413623320'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario7.second_testcase(TestScenario7.java:19)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario4.fifth_testcase(TestScenario4.java:38)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace657736958", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace657736958'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario4.fifth_testcase(TestScenario4.java:38)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario9.forth_testcase(TestScenario9.java:31)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1972326147", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1972326147'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario9.forth_testcase(TestScenario9.java:31)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at testNGDemo.TestScenario4.second_testcase(TestScenario4.java:19)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1345483087", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1345483087'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1590)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1079)
	at org.testng.Assert.assertEquals(Assert.java:1043)
	at org.testng.Assert.assertEquals(Assert.java:1089)
	at testNGDemo.TestScenario4.second_testcase(TestScenario4.java:19)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testNGDemo.TestScenario10.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='org.xyz.auto.saucedemo.KeyboardMouseWindows.handleFrames()'><b>handleFrames</b><br>Test class: org.xyz.auto.saucedemo.KeyboardMouseWindows</td>
<td></td>
<td>7</td>
<td>org.xyz.auto.saucedemo.KeyboardMouseWindows@1fb700ee</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='org.xyz.auto.saucedemo.KeyboardMouseWindows.handleAlerts()'><b>handleAlerts</b><br>Test class: org.xyz.auto.saucedemo.KeyboardMouseWindows</td>
<td></td>
<td>9</td>
<td>org.xyz.auto.saucedemo.KeyboardMouseWindows@1fb700ee</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario@c03cf28</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario8.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario8</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario8@5f683daf</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario9.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario9</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario9@4a668b6e</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.fifth_testcase()'><b>fifth_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.sixth_testcase()'><b>sixth_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario2.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario2</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario2@46fa7c39</td></tr>
<tr>
<td title='testNGDemo.TestScenario3.first_testcase()'><b>first_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
<tr>
<td title='testNGDemo.TestScenario4.third_testcase()'><b>third_testcase</b><br>Test class: testNGDemo.TestScenario4</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario4@1329eff</td></tr>
<tr>
<td title='testNGDemo.TestScenario10.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario10</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario10@4f67eb2a</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.seventh_testcase()'><b>seventh_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
<tr>
<td title='testNGDemo.TestScenario6.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario6</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario6@55f616cf</td></tr>
<tr>
<td title='testNGDemo.TestScenario7.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario7</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario7@5f16132a</td></tr>
<tr>
<td title='testNGDemo.TestScenario5.second_testcase()'><b>second_testcase</b><br>Test class: testNGDemo.TestScenario5</td>
<td></td>
<td>0</td>
<td>testNGDemo.TestScenario5@1356d4d4</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testNGDemo.TestScenario3.forth_testcase()'><b>forth_testcase</b><br>Test class: testNGDemo.TestScenario3</td>
<td><div><pre>java.lang.Throwable: Method testNGDemo.TestScenario3.forth_testcase() on instance testNGDemo.TestScenario3@53aad5d5 depends on not successfully finished methods 
[testNGDemo.TestScenario3.third_testcase() on instance testNGDemo.TestScenario3@53aad5d5]
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1855610584", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1855610584'><pre>java.lang.Throwable: Method testNGDemo.TestScenario3.forth_testcase() on instance testNGDemo.TestScenario3@53aad5d5 depends on not successfully finished methods 
[testNGDemo.TestScenario3.third_testcase() on instance testNGDemo.TestScenario3@53aad5d5]
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>0</td>
<td>testNGDemo.TestScenario3@53aad5d5</td></tr>
</table><p>
</body>
</html>